### ISSUE TITLE
Remote module does not pass remote execution wall time to SpawnResult

### ISSUE THREAD

**Issue by exoson** [bazelbuild/bazel#15283]
### Description of the feature request:

Pass remote execution wall time to the created SpawnResult objects in `src/main/java/com/google/devtools/build/lib/remote/util/Utils.java:createSpawnResult`. PR for this change is https://github.com/bazelbuild/bazel/pull/15282

### What underlying problem are you trying to solve with this feature?

Currently `src/main/java/com/google/devtools/build/lib/remote/util/Utils.java:createSpawnResult` does not set the wall time to the created SpawnResult. In `src/main/java/com/google/devtools/build/lib/exec/StandaloneTestStrategy.java` this wall time would be used to set the test duration if it had been set to the SpawnResult. This causes the test duration to contain misleading times due to using the total time including queue times etc. instead.

### Which operating system are you running Bazel on?

N/A

### What is the output of `bazel info release`?

N/A

### If `bazel info release` returns `development version` or `(@non-git)`, tell us how you built Bazel.

I ran `bazel build //src:bazel`.

### What's the output of `git remote get-url origin; git rev-parse master; git rev-parse HEAD` ?

```text
https://github.com/bazelbuild/bazel.git
6e9239ed904e01ea270f3c7d2e05c5f0be0ee341
6e9239ed904e01ea270f3c7d2e05c5f0be0ee341
```

**Comment by exoson** [comment]
https://github.com/bazelbuild/bazel/pull/15256 seems to fix this issue

**Comment by coeuvre** [comment]
Fixed by 93029d8f1f01be440d8e06cce9585ad912f9169b.
